src/app/content/components/Sidebar/styled/index.tsx

```diff
@@ -11,26 +11,35 @@ import ContentLinkComponent from '../../ContentLink';
 
 export * from './wrapper';
 
-const numberCharacterWidth = 1;
-const numberPeriodWidth = 0.2;
+/* to regenerate these numbers, run this in a rex browser window
+(
+  (element) => 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890.'.split('').reduce(
```

Comment:
Since the code assumes all numbers have the same width, and it seems to use the widest letter it seems clearer to write:

```suggestion
  (element) => 'wW1234567890.'.split('').reduce(
```